pkgs/spark-wallet/verify.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p gnupg
+set -euo pipefail
+
+TMPDIR="$(mktemp -d -p /tmp)"
+trap "rm -rf $TMPDIR" EXIT
+
+version=$(grep spark pkg.json | cut -d^  -f2 | cut -d '"' -f1) # Get only spark-wallet version number from pkg.json
+export GNUPGHOME=$TMPDIR
+gpg --keyserver hkps://keyserver.ubuntu.com --recv-key FCF19B67866562F08A43AAD681F6104CD0F150FC
```

Comment:
`electrs/get-sha256.sh` uses `hkps://hkps.pool.sks-keyservers.net`. We should choose one that we use consistently.

Reply:
This was based on a mistaken belief that shesek's updated key was only available from keyserver.ubuntu.com, fixed in 79785f6